### DISCUSSION TITLE
Bug in Scenario Trends

### DISCUSSION THREAD

**Discussion by jorgetamayo21** [ludeknovy/jtl-reporter#233] [Q&A]
I have 4 executions of a scenario, Scenario Trends orders them from oldest to newest from left to right, but clicking the first one on the left instead of taking me to the oldest takes me to the newest

**Comment by ludeknovy** [comment]
Hi @jorgetamayo21, let me investigate this further.

**Reply by ludeknovy** [reply]
Reproduced and fix prepared.
https://github.com/ludeknovy/jtl-reporter-fe/pull/358

**Reply by ludeknovy** [reply]
Btw, there will be one more fix coming for the label trends charts...

**Reply by ludeknovy** [reply]
https://github.com/ludeknovy/jtl-reporter-fe/pull/359
